protocols/kad/src/behaviour.rs

```diff
@@ -1385,6 +1387,15 @@ where
         }
     }
 
+    /// A new peer has been inserted in the routing table but we check if the routing
+    /// table is currently small (less that `K_VALUE` peers are present) and only
+    /// trigger a bootstrap in that case
+    fn trigger_bootstrap_if_routing_table_is_almost_empty(&mut self) {
```

Comment:
fn name could be more concise (e.g `bootstrap_on_low_peers`)

Reply:
Definitely :+1: it's done.